lib/core/src/lib/dialogs/unsaved-changes-dialog/unsaved-changes-dialog.component.spec.ts

```diff
@@ -106,13 +108,13 @@ describe('UnsavedChangesDialog', () => {
         it('should update storageService to true when checkbox is checked', () => {
             const event = { checked: true };
             savePreferenceCheckbox.triggerEventHandler('change', event);
-            expect(storageServiceMock.setItem).toHaveBeenCalledWith(UnsavedChangesDialogComponent.UNSAVED_CHANGES_MODAL_HIDDEN, 'true');
+            expect(storageServiceMock.setItem).toHaveBeenCalledWith(UNSAVED_CHANGES_KEY, 'true');
```

Comment:
You should not use storage service anymore as it's not used by dialog. Instead please remove mock for storage service and use mocked UserPreferencesService